woocommerce/woocommerce-android
# Update mediapicker dependency to be fetched from S3 and enable composite build for it

### Description

https://github.com/wordpress-mobile/WordPress-MediaPicker-Android/pull/52 starts publishing the `WordPress-MediaPicker-Android` library to our S3 Maven and this PR updates the project so these dependencies will be fetched from the S3 Maven. It also adds composite build support to make it easier to work with the library.

_Note that https://github.com/wordpress-mobile/WordPress-MediaPicker-Android/pull/52 is not reviewed yet. I'll make it available for review once this PR is approved. It's unlikely that any change in that PR will impact this integration._

### Testing instructions

I am not familiar with the media picker, but a smoke test of the library should be enough. It'd be great if it can also be tested with composite build which can be enabled by:
* Copy the `local-builds.gradle-example` file as `local-builds.gradle`
* Uncomment & update the path of the `localMediaPickerPath` to your local `WordPress-MediaPicker-Android`
* Gradle sync & verify that Android Studio shows the `Media Picker` project
* Smoke test the media picker
* Make a change in the media picker library and verify that you can use it immediately in the app
